Doc/library/re.rst

```diff
@@ -632,8 +632,8 @@ character ``'$'``.
    single: \x; in regular expressions
    single: \\; in regular expressions
 
-Most of the standard escapes supported by Python string literals are also
-accepted by the regular expression parser::
+Most of the :ref:`standard escapes <escape-sequences>` supported by Python
```

Comment:
Unless there is a good reason to use the term _standard escapes_, I would prefer to use _escape sequences_, since that seems to be used everywhere else. 

In addition, I would consider adding a heading in reference/lexical_analysis.rst: _Standard escape sequences_ (or just _Escape sequences_), as it seems to be used a lot.

Reply:
Good idea. Done.